2-mandatory/1-writers.js

```diff
@@ -71,7 +74,13 @@ Exercise 2:
 */
 
 function logDeadWritersInTheirForties() {
-  // write your code here
+  
+  const listName = writers.filter( el => el.age >= 40 && el.age < 50 && !el.alive);
```

Comment:
```suggestion
  const listName = writers.filter( ({ age, alive })=> age >= 40 && age < 50 && !alive);
```
We can make the code easier to read by destructuring parameters passed

Reply:
I could learn by reviewing these codes and reading the comments. and also your codes are easy to read.